packages/main/test/specs/Menu.spec.js

```diff
@@ -26,6 +26,7 @@ describe("Menu interaction", () => {
 
 		assert.strictEqual(await menuItems.length, 7, "There are proper count of menu items in the top level menu");
 		assert.strictEqual(await listItems.length, 7, "There are proper count of list items in the top level menu popover list");
+		assert.strictEqual(await listItems[0].getAttribute("additional-text"), await menuItems[0].getAttribute("additional-text"), "The first list item has proper additional text set");
```

Comment:
I thinks there is no test coverage of the other API for hiding it.

Reply:
Yepp. Added test for hideAdditionalText.